proxy/limits/limits.go

```diff
@@ -66,16 +66,23 @@ func SetupFDLimits(wantFDs uint64) error {
 		rlim2.Max = wantFDs
 		rlim2.Cur = wantFDs
 		if err := syscallSetrlimit(syscall.RLIMIT_NOFILE, rlim2); err == nil {
-			logging.Verbosef("Rlimits for file descriptors set to {%v}", rlim2)
+			logging.Verbosef("Rlimits for file descriptors set to {Current = %v, Max = %v}", rlim2.Cur, rlim2.Max)
```

Comment:
Optional flyby: consider writing `%+v` instead of `{ ... }`, it has a similar output (https://play.golang.org/p/YLazOI6dGTu)

Reply:
That's a good idea. I just wish the leading ampersand wasn't included in this case, as it's a user facing message instead of a debug message.